test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpSectionTest/DoesNotRequireSpaceBetweenSectionNameAndOpenBrace.stree.txt

```diff
@@ -20,8 +20,8 @@ Markup block - Gen<None> - 26 - (0:0,0)
                     HtmlTokenType.OpenAngle;[<];
                     HtmlTokenType.Text;[p];
                     HtmlTokenType.CloseAngle;[>];
-            Markup span - Gen<Markup> - [Foo] - SpanEditHandler;Accepts:Any - (17:0,17) - Tokens:1
-                HtmlTokenType.Text;[Foo];
+            SyntaxKind.HtmlText - [Foo] - [17..20) - FullWidth: 3 - Slots: 1
+                SyntaxKind.HtmlTextLiteralToken;[Foo];
```

Comment:
Food for thought: Given that the underlying implementation has a dual state of a single token or having > 1 token this will lead to us special casing understanding of the HtmlText syntax node.

I'm not saying it's wrong, just saying it will make certain decisions based on the syntax tree slightly more complex due to the inconsistency.

Reply:
Yup, this is a valid concern. I am planning to keep going with this until this causes such issues. I would prefer not having a list for single entries as long as it's not too complex to maintain.

Reply:
The syntax tree generally reflects the structure of the syntax